### DISCUSSION TITLE
get_lword and get_lint in a real plc tested

### DISCUSSION THREAD

**Discussion by sivte** [gijzelaerr/python-snap7#433] [Show and tell]
Hello, 
I have tested the "get_lword" and "get_lint" functions (util.py file) in a real plc, specifically in an s7 1511-1 PN.
it works perfectly except for a small change in "get_lword" (the length of a lword is 64 bit not 32).

```
def get_lword(bytearray_: bytearray, byte_index: int) -> bytearray:
    data = bytearray_[byte_index:byte_index + 8]
    dword = struct.unpack('>Q', struct.pack('8B', *data))[0]
    return dword
```

**Comment by gijzelaerr** [comment]
Closing as resolved. Thank you for the real PLC testing confirmation. The `get_lword` fix (using 8 bytes instead of 4) has been incorporated.
